contrib/sync-places.py

```diff
@@ -109,6 +109,13 @@ async def do_sync(session, args):
                     changed = True
 
             tags = config["places"][name].get("tags", {}).copy()
+            for k, v in tags.items():
+                if not isinstance(k, str):
+                    tags[str(k)] = v
+                    del(tags[k])
+                if not isinstance(v, str):
+                    tags[k] = str(v)
+
```

Comment:
Now that the script resides separated in graph-places.py, this should be a separate PR.